lib/manageiq/postgres_ha_admin/server_store.rb

```diff
@@ -22,10 +22,11 @@ def connection_info_list
       end
     end
 
-    def update_servers(connection)
+    def update_servers(connection, handler_name)
```

Comment:
A breaking change... if need be, I can have this be a default value so it won't old callers.

Reply:
if you make it optional, then technically it's not a breaking change?